src/main/resources/application-prod.properties

```diff
@@ -1 +1,6 @@
 server.port=5000
+
+#aws
+cloud.aws.credentials.access-key="AWS_ACCESS_KEY"
+cloud.aws.credentials.secret-key="AWS_SECRET_KEY"
+cloud.aws.region.static="us-east-1"
```

Comment:
make sure there's a new line at the end of this file because we append to it when we write secrets.